RecoLocalTracker/SiPixelRecHits/src/PixelCPEClusterRepair.cc

```diff
@@ -631,15 +627,27 @@ LocalError PixelCPEClusterRepair::localError(DetParam const& theDetParam, Cluste
   //--- (never used, in fact: let comment it out, shut up the complains of the static analyzer, and save a few CPU cycles)
   float xerr = 0.0f, yerr = 0.0f;
 
-  //--- Check status of both template calls.
-  if
-    UNLIKELY((theClusterParam.ierr != 0) || (theClusterParam.ierr2 != 0)) {
+  // Check if the errors were already set at the clusters splitting level
+  if (theClusterParam.theCluster->getSplitClusterErrorX() > 0.0f &&
+      theClusterParam.theCluster->getSplitClusterErrorX() < 7777.7f &&
```

Comment:
What is 7777.7f? If we can, let's try to avoid hardcoded magic numbers. Could `std::numeric_limits::max` work, or otherwise making it a static constexpr in the header?

Reply:
For the record, that number is like that since the dawn of time: https://github.com/cms-cvs-history/RecoLocalTracker-SiPixelRecHits/blob/master/src/PixelCPETemplateReco.cc#L483.

and since AFAIK it's set here:

https://github.com/cms-sw/cmssw/blob/971b61e8ee9899c32c57e629a865b780428a42ec/RecoLocalTracker/SubCollectionProducers/src/JetCoreClusterSplitter.cc#L167-L168

it's not very clear to me from where the `7777.7f` is coming from.  Perhaps for this hotfix, to avoid other unpleasant surprises I would make it `static contexpr` as well and defer to a later PR to find a proper limit.